sass/index.scss

```diff
@@ -234,34 +234,40 @@ section {
 
 // About
 #about {
-    #about-content {
+    #about-cards {
         display: flex;
         justify-content: space-between;
-        align-items: center;
+        margin: 3rem 0rem;
 
-        #about-text {
-            width: 60%;
+        .card {
+            background-color: white;
+            border: 1px solid $light-blue;
+            border-radius: 6px;
+            padding: 20px;
+            width: 45%;
         }
     }
 
-    #about-buttons {
+    #about-stats {
         display: flex;
-        flex-direction: column;
-        margin: 0px auto;
-        align-items: flex-start;
+        justify-content: space-between;
 
-        a {
-            @include button-primary;
-            display: flex;
-            align-items: center;
-            margin: 20px 0px;
-            width: 100%;
+        #stats-content {
+            width: 50%;
+
+            #stats {
+                display: flex;
+                justify-content: space-between;
+                flex-wrap: wrap;
+
+                div {
+                    padding: 20px;
+                }
+            }
```

Comment:
The Figma design shows the leftmost statistic as being left-aligned with the left side of the container, but this line breaks that.

Figma:
![image](https://user-images.githubusercontent.com/16471311/144679287-2430ef47-fd2d-4b5d-b2ac-514f76f4d79d.png)

Current implementation:
![image](https://user-images.githubusercontent.com/16471311/144679440-70de9faf-b91d-4c4f-a567-c27839c4f4ab.png)

Here's a possible fix that seems to work well, but you can test it for yourself and make changes as needed:
```suggestion
            #stats {
                display: flex;
                gap: 20px;
                justify-content: space-between;
                padding: 20px 20px 20px 0;
                flex-wrap: wrap;
            }
```

Reply:
I fixed it a bit better in the next update